cypress/e2e/marc/marc-bibliographic/edit-marc-bib/automated-linking/edit-marc-bib-link-headings-unlink-link-again.cy.js

```diff
@@ -0,0 +1,227 @@
+      InventoryInstances.selectInstance();
+      // #2 Click on "Actions" button in second pane → Select "Edit MARC bibliographic record" option
+      InventoryInstance.editMarcBibliographicRecord();
+      QuickMarcEditor.verifyEnabledLinkHeadingsButton();
+      // #3 Click on the "Link headings" button.
+      QuickMarcEditor.clickLinkHeadingsButton();
+      QuickMarcEditor.checkCallout(
+        'Field 100, 600, and 650 has been linked to MARC authority record(s).',
+      );
+      linkedFields.forEach((field) => QuickMarcEditor.verifyRowLinked(field.rowIndex, true));
+      QuickMarcEditor.verifyDisabledLinkHeadingsButton();
+      // #4 Unlink 4 automatically linked fields by clicking on "Unlink from MARC authority record" icon.
+      linkedFields.forEach((field) => {
+        if (field.isUnlinked) {
+          QuickMarcEditor.clickUnlinkIconInTagField(field.rowIndex);
+          cy.wait(1000);
+          QuickMarcEditor.confirmUnlinkingField();
+          QuickMarcEditor.verifyRowLinked(field.rowIndex, false);
+        }
+      });
+      QuickMarcEditor.verifyEnabledLinkHeadingsButton();
+      // #5 Link first unlinked field to different "MARC authority" record ("$0" value should be changed). For example: link 100 to "n83169267"
+      QuickMarcEditor.clickLinkIconInTagField(linkedFields[0].rowIndex);
+      InventoryInstance.verifySelectMarcAuthorityModal();
+      MarcAuthorities.switchToSearch();
+      InventoryInstance.searchResults('n83169267');
+      MarcAuthoritiesSearch.selectAuthorityByIndex(0);
+      InventoryInstance.clickLinkButton();
+      QuickMarcEditor.verifyTagFieldAfterLinking(
+        linkedFields[0].rowIndex,
+        linkedFields[0].tag,
+        '1',
+        '\\',
+        '$a Lee, Stan, $d 1922-2018',
+        '$e author.',
+        '$0 id.loc.gov/authorities/names/n83169267',
+        '',
+      );
+      // #6 Edit subfield "$0" value of unlinked field to another valid (matched with "naturalId" of existing "MARC authority" record)
+      QuickMarcEditor.updateExistingFieldContent(
+        linkedFields[3].rowIndex,
+        linkedFields[3].newContent,
+      );
+      // #7 Delete subfield "$0" of unlinked field.
+      QuickMarcEditor.updateExistingFieldContent(
+        linkedFields[4].rowIndex,
+        linkedFields[4].newContent,
+      );
+      // #8 Click on the "Link headings" button again.
+      QuickMarcEditor.clickLinkHeadingsButton();
+      QuickMarcEditor.checkCallout(
+        'Field 600 and 650 has been linked to MARC authority record(s).',
+      );
+      QuickMarcEditor.verifyRowLinked(linkedFields[0].rowIndex, true);
+      QuickMarcEditor.verifyRowLinked(linkedFields[3].rowIndex, true);
+      QuickMarcEditor.verifyDisabledLinkHeadingsButton();
+      // #9 Click on the "Save & keep editing" button.
+      QuickMarcEditor.clickSaveAndKeepEditing();
+      linkedFields.forEach((field, index) => {
+        if (index !== 4) {
```

Comment:
not clear what we verify here. Magic numbers

Reply:
Fixed